src/map/map.ts

```diff
@@ -594,13 +596,33 @@ export class PropertiesMap {
             });
         }
 
+        var default_presets : { [preset: string]: any; }  = {
+          "color" : (properties.length>=3 ? properties[2] : '' ),
+          "x-axis" : properties[0],
+          "y-axis" : properties[1],
+          "z-axis" : '',
+          "palette" : 'inferno'
+        };
+
+        if ( presets !== undefined ) {
+            console.log("setupSettings presets");
+            console.log(presets)
+            console.log("color", presets["color"])
+            for ( const preset in default_presets) {
+                console.log(preset, " >> ", presets[preset])
+                if ( preset in presets) {
+                    default_presets[preset] = presets[preset];
```

Comment:
This is were we should validate that the preset/user-provided settings correspond to an existing property in the dataset